metricbeat/module/haproxy/_meta/haproxy.conf

```diff
@@ -0,0 +1,39 @@
+
+global
+  # maximum number of simultaneous active connections
+  maxconn 4000
+  #daemon
+  user  root
+  group staff
+
+  # for restarts
+  pidfile /var/run/haproxy.pid
+  # Logging to syslog facility local0
+  log   127.0.0.1       local0
+  stats socket 0.0.0.0:14567
```

Comment:
Is this something that is enabled by default in most haproxy setups? If not, could be worth to mention this in the haproxy module docs.